gym/envs/robot_locomotion_group/shoe/shoe_env.py

```diff
@@ -0,0 +1,41 @@
+from future.utils import iteritems
+import meshcat
+import numpy as np
+import os
+import yaml
+
+import gym
+from gym.envs.robot_locomotion_group.shoe.build_shoe_diagram import (
+    reset_simulator_from_config,
+    build_shoe_diagram
+)
+class ShoeEnv(gym.Env):
+    def __init__(self, config=None):
+        if config is None:
+            shoe_dir = os.path.dirname(os.path.abspath(__file__))
+            config_path = os.path.join(shoe_dir, 'config.yaml')
+            self.config = yaml.safe_load(open(config_path, 'r'))
+        else:
+            self.config = config
+        if self.config["env"]["meshcat"]:
+            vis = meshcat.Visualizer()
+        self.step_dt = self.config["env"]["step_dt"]
+        sim_objects = build_shoe_diagram(self.config)
+        self.simulator, self.diagram, self.systems, self.visualizer = sim_objects
```

Comment:
I think things are working nice here.  

Would it be possible to implement the definitions of 
`self.action_space` and `self.observation_space` as an instance of `gym.spaces.Box`? 

Example here: https://github.com/RobotLocomotion/gym/blob/4d8d08c15596acf7bab540c65564545ea2a2307c/gym/envs/robot_locomotion_group/carrot/carrot_env.py#L15

In principle we should be able to run the environment using 
`python3 examples/agents/random_agent.py Shoe-v0`,

but it seems like it currently doesn't run because the spaces are not implemented.